content/applications/sales/sales/send_quotations/get_paid_to_validate.rst

```diff
@@ -1,53 +1,68 @@
-useful and efficient for international transactions. With Odoo Sales, you can use online payments
-to get automatic orders confirmation. Online payments are made instantly, so it's very convenient
-and saves lots of time in a basic sales process.
+With Odoo *Sales*, online payments can be used to get automatic confirmation on orders.
 
 Enable online payment
 =====================
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Online Payment**
-feature.
+Go to :menuselection:`Sales app --> Configuration --> Settings`, scroll to the
+:guilabel:`Quotations & Orders` heading, and activate the :guilabel:`Online Payment` feature by
+checking the box next to it, and clicking :guilabel:`Save`.
 
-.. image:: get_paid_to_validate/get_paid_1.png
+.. image:: get_paid_to_validate/online-payment-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable online payment on Odoo Sales?
+   :alt: How to enable online payment on Odoo Sales.
 
-There, you will have direct access to the **Payment Acquirers** page. It will allow you to select
-and configure your acquirers of choice. Before creating or modifying a payment acquirer, be sure to
-check out our documentation about how to be paid with payment acquirers such as
-:doc:`/applications/finance/payment_acquirers/paypal`,
-:doc:`/applications/finance/payment_acquirers/authorize`, and others in the
-:doc:`/applications/finance/payment_acquirers` documentation.
+After checking the box beside the :guilabel:`Online Payment` feature, a link to the
+:guilabel:`Payment Acquirers` appears beneath it. Click that link to reveal the entire
+:guilabel:`Payment Acquirers` page.
+
+.. image:: get_paid_to_validate/payment-acquirers.png
+   :align: center
+   :alt: Payment acquirers page in the Odoo Sales application.
+
+On the :guilabel:`Payment Acquirers` page, payment acquirers can be selected and configured. Before
+creating (or modifying) a payment acquirer, be sure to review the documentation related to the ways
+payment acquirers can be used with Odoo, such as:
+
+- :doc:`/applications/finance/payment_acquirers/paypal`
+- :doc:`/applications/finance/payment_acquirers/authorize`
+- :doc:`/applications/finance/payment_acquirers`
 
 .. note::
-   If you are familiar with this documentation: :doc:`quote_template`; you
-   can activate or not the **Online Payment** feature for each template you use, under their
-   confirmation tab.
+   On quotation templates, the :guilabel:`Online Payment` feature is located under the
+   :guilabel:`Confirmation` tab.
+
+   .. image:: get_paid_to_validate/confirmation-tab.png
+      :align: center
+      :alt: Confirmation tab with the online payment feature.
+
+   On standard quotations, the :guilabel:`Online Payment` feature is located under the
+   :guilabel:`Other Info` tab.
+
+   .. image:: get_paid_to_validate/other-info.png
+      :align: center
+      :alt: Other Info tab with the online payment feature.
```

Comment:
These two images are not entirely necessary, I think your text is sufficient enough. Can remove these two if you agree :) (don't forget to `git rm` the media files)
```suggestion
   On quotation templates, the :guilabel:`Online Payment` feature is located under the
   :guilabel:`Confirmation` tab.

   On standard quotations, the :guilabel:`Online Payment` feature is located under the
   :guilabel:`Other Info` tab.
```